faiss_vector_posting.go

```diff
@@ -318,23 +348,61 @@ func (sb *SegmentBase) InterpretVectorIndex(field string, except *roaring.Bitmap
 					return rv, nil
 				}
 
-				scores, ids, err := vecIndex.SearchWithoutIDs(qVector, k, vectorIDsToExclude, params)
+				var scores []float32
+				var ids []int64
```

Comment:
i think these 3 three vars can be init'd as part of 355 itself, since there isn't an if condition around SearchWithoutIDs()